### PR TITLE
Adding Grove Zero blog post

### PR DESCRIPTION
**Pull request by riknoll**

### CODE REVIEW

File: docs/blog/seeed/v0.2.10.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+![Seeed Grove Zero](/static/blog/seeed/SeeedTarget.png)
+# Microsoft MakeCode for Seeed Grove Zero
+
+**Posted on November 14, 2017 by [riknoll](https://github.com/riknoll)**
+
+The Microsoft MakeCode Team is excited to announce the beta release of an editor for the [Seeed Grove Zero](https://www.seeedstudio.com/edu/grove-zero.html) system!
+
+Grove Zero is made up of magnetic modules that snap together for endless possibilities. Program
+the main board to control all of the various inputs and outputs out of the box. The magnetic connectors
```
Comment (microsoftsam) [inline docs/blog/seeed/v0.2.10.md]: Are the new lines here intended? Would this break up the line in markdown?

Comment (riknoll) [reply docs/blog/seeed/v0.2.10.md]: Yep, intended to make the .md more readable. Markdown requires a double new line for a new line to show up in the rendered page.

Comment (microsoftsam) [reply docs/blog/seeed/v0.2.10.md]: Ah makes sense :)

File: docs/blog/seeed/v0.2.10.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+![Seeed Grove Zero](/static/blog/seeed/SeeedTarget.png)
+# Microsoft MakeCode for Seeed Grove Zero
```
Comment (pelikhan) [inline docs/blog/seeed/v0.2.10.md]: Add beta in title

**Comment by abchatra** [comment]
👍

**Pull request merged**